### PR TITLE
Revert "Temporarily remove Support for FIPS cryptography topic"

### PR DESCRIPTION
**Pull request by mjpytlak**
This reverts commit d4f72653f9d28b6c0fb3e42d0b68677877fc1ec8.

Version(s):
4.13+

Issue:
This issue addresses [osdocs-7806](https://issues.redhat.com/browse/OSDOCS-7806)

Link to docs preview:

- [Support for FIPS cryptography](https://64917--docspreview.netlify.app/openshift-enterprise/latest/installing/installing-fips)
- Customizing Nodes > [Additional information](https://64917--docspreview.netlify.app/openshift-enterprise/latest/installing/install_config/installing-customizing#additional-resources) link
- [Do you need extra security for your cluster?](https://64917--docspreview.netlify.app/openshift-enterprise/latest/installing/installing-preparing#installing-preparing-security) (last sentence + admonition)
- [Configuring FIPS through the Agent-based installer](https://64917--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer#agent-installer-configuring-fips-compliance_preparing-to-install-with-agent-based-installer) (Additional resources link)
- [Operator metadata annotations](https://64917--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-generating-csvs#osdk-csv-manual-annotations_osdk-generating-csvs) (Additional resources link)
- [Understanding compliance and risk management](https://64917--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-generating-csvs#osdk-csv-manual-annotations_osdk-generating-csvs) (Additional resources link)
- [Hardening after the cluster is running](https://64917--docspreview.netlify.app/openshift-enterprise/latest/security/container_security/security-hardening#security-harden-after-installation_security-hardening) (Additional resources link)
- [Understanding host and VM security](https://64917--docspreview.netlify.app/openshift-enterprise/latest/security/container_security/security-hosts-vms) (Additional resources link)
- [Preparing your cluster for OpenShift Virtualization](https://64917--docspreview.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt) (sentence below admonition)

QE review:
- [N/A] QE has approved this change.
Reverting the changes that were made as part of https://github.com/openshift/openshift-docs/pull/60826.

Additional information:

### CODE REVIEW

File: installing/installing-preparing.adoc [hunk 1]
```diff
@@ -77,12 +77,11 @@ If you use a user-provisioned installation method, you can configure a proxy for
...
...phere/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[vSphere], or xref:../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc#installing-restricted-networks-bare-metal[bare metal]. You can also install a cluster into a restricted network using installer-provisioned infrastructure by following detailed instructions for xref:../installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc#installing-restricted-networks-aws-installer-provisioned[AWS], xref:../installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc#installing-restricted-networks-gcp-installer-provisioned[GCP], xref:../installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.adoc#installing-restricted-networks-nutanix-installer-provisioned[Nutanix, xref:../installing/installing_openstack/installing-openstack-installer-restricted.adoc#installing-openstack-installer-restricted[{rh-openstack}], and xref:../installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc#installing-restricted-networks-installer-provisioned-vsphere[vSphere].
```
Comment (mjpytlak) [inline installing/installing-preparing.adoc:80]: Worth noting that the broken xref `[Nutanix, xerf:...` was not introduced by Ashley's PR or the subsequent revert. This was introduced by https://github.com/openshift/openshift-docs/pull/60438/files#diff-7a06762359d4d38228c7c8d974a10016f2e10f4230a3ff910b1b5b6818514c07R79.

I happened to see the broken link when looking at preview build and fixed it here.

**Comment by ahardin-rh** [comment]
LGTM

**Comment by kalexand-rh** [comment]
> @kalexand-rh Wanted to reach out on this one. As part of effort around https://issues.redhat.com/browse/OSDOCS-7747, I need to revert changes @ahardin-rh made when we first began to remove FIPS-related content from 4.13.
> 
> I ended up using `git revert <commitID>` to revert the changes Ashley made. By all accounts, the files look good. Just want to make sure I handled this correctly from a process perspective.
> 
> Once I get the preview links in, I would appreciate it if either of you could approve from a peer review perspective.

That's correct from a process perspective. It should be good to merge.

**Comment by mjpytlak** [comment]
Worth noting that while tabled for 4.13, I am holding off on that cherry pick until I am ready to completely reintroduce FIPS docs to 4.13. For the time being, the plan is to get 4.14 updates with the new language and then update 4.13.

**Pull request merged**